isolated-contexts.bs

```diff
@@ -411,18 +410,15 @@ these properties will not mutate during an environment's lifetime.
 <div algorithm="environment settings object is an isolated context">
 An [=environment settings object=] |environment| is an
 <dfn export>isolated context</dfn> if the following algorithm returns `true`:
-    1.  Let |browsing context group| be the [=browsing context group=] that 
-        |environment| belongs to.
     1.  If |environment| does not [=environment settings object/meaningfully
         mitigate injection attacks=], return `false`.
-    1.  If |browsing context group|'s [=cross-origin isolated capability=] is
+    1.  If |environment|'s [=cross-origin isolated capability=] is
         not [=concrete=], return `false`.
     1.  If |environment| does not [=environment settings object/mitigate UI
         Redressing attacks=], return `false`.
-    1.  If |browsing context group|'s [=browsing context group/integrity
-        origin=] is null, return `false`.
-    1.  If |environment|'s [=origin=] is not equal to [=browsing context group/
-        integrity origin=], return `false`.
+    1.  Let |origin| be |environment|'s [=origin=].
+    1.  If the [=user agent=]'s [=origin integrity verification map=][|origin|]
+        does not [=exist=], return `false`.
```

Comment:
You can use `[=map/exist=]` to avoid needing to specify the right definition in the `link-defaults` section.
```suggestion
        does not [=map/exist=], return `false`.
```

Reply:
Thanks, done